src/allotropy/allotrope/models/cell_counting_benchling_2023_11_cell_counting.py

```diff
@@ -376,11 +376,6 @@ class DeviceControlDocumentItem:
     detector_wavelength_setting: Optional[TQuantityValueNanometer] = None
 
 
-@dataclass
-class Asm:
```

Comment:
Note to reviewer: this is an example of a harmless, but confusing pattern that the schema cleaner removes. 

Previously, the generation script generated a class based on the template, with an optional manifest field, and then the main model specified the asm manifest as required, causing the model class to both have the field, and inherit from this model.

The cleaner removes the extra class.